tests/digital/test_digital_tjm.py

```
@@ -56,76 +45,174 @@
 
-def _phase_align(reference: np.ndarray, state: np.ndarray) -> np.ndarray:
-    """Align ``state``'s global phase to ``reference``.
+# --- create_local_noise_model ---
 
-    Args:
-        reference: Reference vector to align against.
-        state: State vector to be phase-aligned.
 
-    Returns:
-        ``state`` with global phase aligned to ``reference``.
+def test_create_local_noise_model() -> None:
```

Ruff (0.15.15)

[warning] Too many branches (15 > 12)

(PLR0912)